docs/conf.py

```diff
@@ -211,7 +211,7 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'xonsh.tex', u'xonsh documentation',
+  ('index', 'xonsh.tex', u'rickshaw documentation',
```

Comment:
Should the 2nd entry still be `xonsh.tex`?

Reply:
Whoops! fixed!